src/teaching-element/Reflection.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <assessment
+    v-bind="$attrs"
+    @assessmentSubmit="data => $emit('assessmentSubmit', data)"
+    @reflectionSubmit="data => $emit('reflectionSubmit', data)"
+    isReflection/>
+</template>
+
+<script>
+import Assessment from './Assessment';
+
+export default {
+  name: 'te-reflection',
+  components: { Assessment }
+};
```

Comment:
It might be ideal candidate for functional component. It doesn't have a state and neither the template...